docs/source/processing_at_scale/processing_with_aws.rst

```diff
@@ -0,0 +1,351 @@
+Click *Create Role* we've called ours `spacetx-batch-uploader`. From the list of available services to prevision your role with select *batch*. Then click through the rest of the wizard
+using the default settings and create the role.
+
+We also need to give this role read and write access to our newly created s3 bucket. To do this we make a new policy and attach it to the `spacetx-batch-uploader` role.
+
+Select *policies* form the left hand panel and click *create policy*. Click on the JSON editor and paste in the following code:
+
+
+::
+
+    {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Sid": "ListObjectsInBucket",
+                "Effect": "Allow",
+                "Action": [
+                    "s3:ListBucket"
+                ],
+                "Resource": [
+                    "arn:aws:s3:::<YOUR BUCKET>"
+                ]
+            },
+            {
+                "Sid": "AllObjectActions",
+                "Effect": "Allow",
+                "Action": "s3:*Object",
+                "Resource": [
+                    "arn:aws:s3:::<YOUR BUCKET>/*"
+                ]
+            }
+        ]
+    }
+
+
+Here's what ours looks like:
+
+::
+
+    {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Sid": "ListObjectsInBucket",
+                "Effect": "Allow",
+                "Action": [
+                    "s3:ListBucket"
+                ],
+                "Resource": [
+                    "arn:aws:s3:::aws-processing-example"
+                ]
+            },
+            {
+                "Sid": "AllObjectActions",
+                "Effect": "Allow",
+                "Action": "s3:*Object",
+                "Resource": [
+                    "arn:aws:s3:::aws-processing-example/*"
+                ]
+            }
+        ]
+    }
+
+
+Name your policy and save it, we named our `spacetx-batch-uploader`. Now navigate back to your new role and attach your s3 uploader policy.
+Our `spactx-batch` role summery now looks like this:
+
+.. figure:: images/aws-tutorial-figure2.png
+   :align: center
+
+Note the ARN of your new role (circled in the image). You'll need it in the next few steps.
+
+
+Register your Jobs
++++++++++++++++++++
+
+Follow the `Getting Started Guide <http://docs.aws.amazon.com/batch/latest/userguide/Batch_GetStarted.html>`_ and ensure you have a valid job queue and compute environment. For this tutorial
+we used the default parameters (our job queue is still called first-run-job-queue).
+
+Here's what out Batch Dashboard looks like:
+
+.. figure:: images/aws-tutorial-figure5.png
+   :align: center
+
+Alright now it's time to register our batch jobs. From the template files open up the file named `register-process-fov-job.json`. This file describes a batch job that will create an ec2 instance using the docker container `spacetx/process-fov`
```

Comment:
Is this a docker container we have to maintain?

Reply:
maybe? It's pretty bare bones though.

Reply:
will it automatically pick up the latest version of starfish?

Reply:
That's a good point.  To keep things up to date we'd need to rebuild them with each new release https://github.com/spacetx/starfish-workflows/blob/master/aws/merge-results-job-docker-files/Dockerfile. Or install starfish within the container

Reply:
We could just add this to our release workflow.